src/ralph/admin/widgets.py

```diff
@@ -19,3 +23,68 @@ def media(self):
     def render(self, name, value, attrs=None):
         attrs['class'] = 'datepicker'
         return super(AdminDateWidget, self).render(name, value, attrs=attrs)
+
+
+class PermissionsSelectWidget(forms.Widget):
+    @property
+    def media(self):
+        js = ['multi.js']
+        return forms.Media(js=[static('src/js/%s' % path) for path in js])
+
+    def value_from_datadict(self, data, files, name):
+        values = data.get(name, None)
+        if not values:
+            return
+        return values.split(',')
+
+    def render(self, name, value, attrs=None, choices=()):
+        final_attrs = self.build_attrs(
+            attrs, type='hidden', name=name, value=','.join(map(str, value)),
+        )
+        return mark_safe(
+            '<ul class="accordion" data-multi="{}" data-accordion>{}</ul>'
+            '<input{} />'.format(
+                name, self.render_options(choices, value), flatatt(final_attrs)
+            )
+        )
+
+    def render_option(self, selected_choices, option_value, option_label):
+        input_id = 'id_option_{}'.format(option_value)
+        attrs = {'id': input_id, 'type': 'checkbox', 'value': option_value}
+        if option_value in selected_choices:
+            attrs['checked'] = 'checked'
+        attrs = self.build_attrs(**attrs)
+        return '<input{}><label{}>{}</label>'.format(
+            flatatt(attrs), flatatt({'for': input_id}), option_label
+        )
+
+    def render_options(self, choices, selected_choices):
+        self.choices = sorted(self.choices, key=lambda x: x[1].split(' | ')[1])
```

Comment:
i would make ' | ' const or anything else that is defined in one place and use everywhere ;)